Identity provider updates in v1.0 and beta (#13660)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/identityproviderbase.md

For Azure AD B2B scenarios , the identity provider is either a [socialIdentityProvider](../resources/socialidentityprovider.md) or a [builtinIdentityProvider](../resources/builtinidentityprovider.md). Configuring an identity provider in your Azure AD tenant enables new Azure AD B2B guest scenarios. For example, an organization with resources in Microsoft 365 can share them with a Gmail user. The Gmail user will use their Google account credentials to authenticate and access the documents.

In an Azure AD B2C directory, the identity provider type is a [socialIdentityProvider](../resources/socialidentityprovider.md). Configuring an identity provider in your Azure AD B2C directory enables users to sign up and sign in using a social account in an application. For example, an application can use Azure AD B2C to allow users to sign up for the service using a Facebook account.

## Methods

| Method       | Return Type  |Description|
|:---------------|:--------|:----------|

> |[List](../api/identitycontainer-list-identityproviders.md)|identityProviderBase collection|Retrieve all identity providers configured in a tenant.|

Reviewer: ```suggestion
|[List](../api/identitycontainer-list-identityproviders.md)|[identityProviderBase](../resources/identityproviderbase.md) collection|Retrieve all identity providers configured in a tenant.|
```

> Represents identity providers with [External Identities](/azure/active-directory/external-identities/) for both Azure Active Directory tenant and an Azure AD B2C tenant.
> 
> For Azure AD B2B scenarios in an Azure AD directory, the identity provider can be a [socialIdentityProvider](../resources/socialidentityprovider.md) or a [builtinIdentityProvider](../resources/builtinidentityprovider.md), which will inherit from identityProviderBase resource type.
> 
> Configuring an identity provider in your Azure AD directory enables new Azure AD B2B guest scenarios. For example, an organization has resources in Microsoft 365 that need to be shared with a Gmail user. The Gmail user will use their Google account credentials to authenticate and access the documents.
> 
> In an Azure AD B2C directory, the identity provider type can be a [socialIdentityProviders](../resources/socialidentityprovider.md) which will inherit from identityProviderBase resource type.
> 
> Configuring an identity provider in your Azure AD B2C directory enables users to sign up and sign in using a social account in an application. For example, an application can use Azure AD B2C to allow users to sign up for the service using a Facebook account.
> 

Reviewer: Recommended change. Could also apply it to the beta topic?
```suggestion
Represents identity providers with [External Identities](/azure/active-directory/external-identities/) for both Azure Active Directory (Azure AD) and Azure AD B2C tenants. It is an abstract type that is inherited by [socialIdentityProvider](../resources/socialidentityprovider.md) and [builtinIdentityProvider](../resources/builtinidentityprovider.md).

For Azure AD B2B scenarios , the identity provider is either a [socialIdentityProvider](../resources/socialidentityprovider.md) or a [builtinIdentityProvider](../resources/builtinidentityprovider.md). Configuring an identity provider in your Azure AD tenant enables new Azure AD B2B guest scenarios. For example, an organization with resources in Microsoft 365 can share them with a Gmail user. The Gmail user will use their Google account credentials to authenticate and access the documents.

In an Azure AD B2C directory, the identity provider type is a [socialIdentityProvider](../resources/socialidentityprovider.md). Configuring an identity provider in your Azure AD B2C directory enables users to sign up and sign in using a social account in an application. For example, an application can use Azure AD B2C to allow users to sign up for the service using a Facebook account.

```

> ## Methods

Reviewer: Always link to the return type topic.
